addons/website_slides/static/src/xml/activity.xml

```diff
@@ -19,4 +19,20 @@
             </t>
         </t>
     </t>
-</templates>
\ No newline at end of file
+
+    <t t-inherit="mail.Activity" t-inherit-mode="extension">
+        <xpath expr="//*[@name='tools']" position="replace">
+            <t t-if="activity.creatorPartner and activity.thread.model === 'slide.channel'">
+                <div name="tools" class="o_ActivityList">
```

Comment:
What is `o_ActivityList`? I think keep class `o_Activity_tools`, and don't need to reintroduce the `name=` here.